sites/all/modules/custom/da_malik/da_malik.install

```diff
@@ -1,8 +1,41 @@
 <?php
+/**
+* Implements Hook enable
+*/
 
+function da_malik_enable() {
+
+	/*setting the homepage*/
```

Comment:
This is nitpicking, and this won't fail for our purposes, but just so that you are aware:
Drupal has specific coding standards. This would fail a test on drupal.org due to:
1) Use 2 spaces to indent, rather than tabs
2) For single line comments, prepend line with // rather than wrapping it with /**/
Worth looking through once in a while: https://www.drupal.org/coding-standards